15/umbraco-commerce/how-to-guides/create-order-via-code.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Learn how to create an order via code in Umbraco Commerce.
+---
+
+# Create an Order via Code
+
+It is sometimes necessary, such as when importing orders from another system, to create a fully finalized order via code. 

```

Comment:
```suggestion
In some cases, such as when importing orders from another system, it may be necessary to create a fully finalized order via code. 
```